presto-record-decoder/src/test/java/io/prestosql/decoder/avro/TestAvroDecoder.java

```diff
@@ -77,7 +77,7 @@
     private static final AvroRowDecoderFactory DECODER_FACTORY = new AvroRowDecoderFactory();
 
     private static final Metadata METADATA = createTestMetadataManager();
-    private static final Type VACHAR_MAP_TYPE = METADATA.getType(parseTypeSignature("map(varchar,varchar)"));
```

Comment:
Please fix this typo in a separate commit as well.

Reply:
Done